### PR TITLE
Fix database commands, allow OTDs in PMs.

### PR DESCRIPTION
**Pull request by HarmanPC**

### CODE REVIEW

**Comment by HarmanPC** [comment]
im hella confused ;;

File: plugins/writing.js [hunk 1]
```diff
@@ -2034,46 +2037,44 @@ let commands = {
...
 				database.myths.pending = null;
 				Storage.exportDatabase('writing');
-			} else if (targets[1] === "delete" && database.myths.pendingDelete !== -1) {
-				database.myths.db.splice(database.myths.pendingDelete, 1);
-				for (let i = database.myths.pendingDelete; i < database.myths.db.length; i++) {
-					database.myths.db[i].id = i;
-					database.myths.lastID = i;
-				}
+			}
+		} else if (targets[1] === "delete" && database.myths.pendingDelete !== -1) {
```
Comment (sirDonovan) [inline plugins/writing.js]: Why did you move this?

Comment (HarmanPC) [reply plugins/writing.js]: Uh I thought it's a extra tab made by me, later.

Comment (HarmanPC) [reply plugins/writing.js]: When I looked at all commands of ;dB I saw it as too away so I pulled one tab backward
